WordPress/Classes/Networking/SiteManagementServiceRemote.swift

```diff
@@ -51,11 +51,11 @@ public class SiteManagementServiceRemote : ServiceRemoteREST
     ///
     public func exportContent(siteID: NSNumber, success: (() -> Void)?, failure: (NSError -> Void)?) {
         let endpoint = "sites/\(siteID)/exports/start"
-        let path = self.pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+        let path = self.pathForEndpoint(endpoint, withVersion: ._1_1)
```

Comment:
That `._1_1` looks really ugly in Swift, but not sure what's the best alternative. The one I'm liking more is to add another value to the enum that doesn't match the prefix so the generated Swift value is `Version_1_1`:

```obj-c
typedef NS_ENUM(NSInteger, ServiceRemoteWordPressComRESTApiVersion)
{
    // Don't use this value
    ServiceRemoteWordPressComRESTApiUnknownVersion = 0,
    ServiceRemoteWordPressComRESTApiVersion_1_1 = 1001,
    ServiceRemoteWordPressComRESTApiVersion_1_2 = 1002,
};
```

https://gist.github.com/koke/0b6d994aaf85bd8239524258b842fea3